x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/analysis/EnrichResolution.java

```diff
@@ -34,11 +35,17 @@ public Collection<ResolvedEnrichPolicy> resolvedEnrichPolicies() {
 
     }
 
+    // created for testing
+    public boolean hasErrors() {
+        return errors.size() > 0;
```

Comment:
nit: can rewrite as `!errors.isEmpty()`.

Reply:
Well, in Elasticsearch that would be `errors.isEmpty() == false` which is more verbose.